### ISSUE TITLE
Move Dimension/Measure type from description

### ISSUE THREAD

**Issue by norbertbede** [pyrooka/metabase-cubejs-driver#64]
**Is your feature request related to a problem? Please describe.**
We need description field for serious dimension and measure description explain users useful informations. ATM description is used for differentiate 

**Describe the solution you'd like**
Opt1. extend table metabase_field column name eg. 

- dimension.Warehouse.linecount
- measure.Warehouse.docdate

Opt2. extend field metabase_field.fingertip column
With a small luck json should be extendable and not dis-function metabase basic functionality.

**Notes**
ATM i disagree with issue: Idea: store the measures only in metrics #50
reason: we are dealing with get work basic measures for simple calculation and/or relative offset of measure - eg. get past month for same measure. (custom metric) - i will explain it soon in another issue.

**Comment by pyrooka** [comment]
Yes, I have already thought about it. As far as I remember, I tried once to add this information to a custom field but it didn't work.
So we have to investigate where can we store the details without breaking the Metabase.

**Comment by norbertbede** [comment]
did you try opt1 or opt2 - described above ? 
can you elaborate to describe you approach ?

**Comment by pyrooka** [comment]
I tried to add an extra field to the Field model, but that made it invalid.
The first one could work but I don't really like the idea to "pollute" the name of the field with the type. In this case, we should leave the field type in the description. For example: "measure; This is the field comment.". But store this info in a separate field, hidden from the user would be the best solution in my opinion.

**Comment by pyrooka** [comment]
@norbertbede what do you think #70 ?

**Comment by norbertbede** [comment]
for now, can be acceptable. the question is what happening when admin forget or make mistake field content change.

**Comment by pyrooka** [comment]
You mean the admin changes the field comment/description?
